pkg/agent/openflow/client.go

```diff
@@ -39,6 +39,12 @@ import (
 
 const maxRetryForOFSwitch = 5
 
+const (
+	TrafficControlFlowPriorityHigh   = "high"
+	TrafficControlFlowPriorityMedium = "medium"
+	TrafficControlFlowPriorityLow    = "low"
+)
```

Comment:
you could define a dedicated type for this:
```go
type TrafficControlFlowPriority string

const (
	TrafficControlFlowPriorityHigh   TrafficControlFlowPriority = "high"
	TrafficControlFlowPriorityMedium TrafficControlFlowPriority = "medium"
	TrafficControlFlowPriorityLow    TrafficControlFlowPriority = "low"
)

func (p TrafficControlFlowPriority) toOFPriority() uint16 {
	switch p {
	case TrafficControlFlowPriorityHigh:
		return priorityHigh
	case TrafficControlFlowPriorityMedium:
		return priorityNormal
	case TrafficControlFlowPriorityLow:
		return priorityLow
	default:
		return 0
	}
}
```

Reply:
Good idea! Thanks